app/src/main/java/com/example/lunark/AccountScreen.java

```diff
@@ -3,40 +3,83 @@
 import android.content.DialogInterface;
 import android.content.Intent;
 import android.os.Bundle;
+import android.view.MenuItem;
 import android.view.View;
+import android.widget.Toast;
 
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.ActionBarDrawerToggle;
 import androidx.appcompat.app.AlertDialog;
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.drawerlayout.widget.DrawerLayout;
+
+import com.google.android.material.navigation.NavigationView;
 
 public class AccountScreen extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_account_screen);
+        setContentView(R.layout.activity_account);
+
+        DrawerLayout drawerLayout = findViewById(R.id.drawer_layout);
+        NavigationView navigationView = findViewById(R.id.nav_view);
+
+        Toolbar toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        ActionBarDrawerToggle actionBarDrawerToggle = new ActionBarDrawerToggle(
+                this, drawerLayout, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close);
+        drawerLayout.addDrawerListener(actionBarDrawerToggle);
+        actionBarDrawerToggle.syncState();
+        navigationView.setNavigationItemSelectedListener(new NavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                int itemId = item.getItemId();
+
+                if (itemId == R.id.menu_home) {
+                    if (!isCurrentActivity(HomeActivity.class)) {
+                        startActivity(new Intent(AccountScreen.this, HomeActivity.class));
+                    }
+                } else if (itemId == R.id.menu_account) {
+                    if (!isCurrentActivity(AccountScreen.class)) {
+                        startActivity(new Intent(AccountScreen.this, AccountScreen.class));
+                    }
+                } else if (itemId == R.id.menu_reservations || itemId == R.id.menu_notifications) {
+                    Toast.makeText(AccountScreen.this, "Screen not implemented", Toast.LENGTH_SHORT).show();
+                } else if (itemId == R.id.menu_logout) {
+                    if (!isCurrentActivity(LoginScreenActivity.class)) {
+                        Intent intent = new Intent(AccountScreen.this, LoginScreenActivity.class);
+                        startActivity(intent);
+                    }
+                }
+
+                drawerLayout.closeDrawers();
+                return true;
```

Comment:
I agree with @vradovic that we should use fragments instead of activities for screens. If we used fragments, this snippet of code would never need to be written. We would just a navigation controller in the home activity and the controller would do all the work for us. This would speed up development since we won't have to repeat and track snippets like this one.
Since the first checkpoint is drawing close, I would recommend leaving this be for now and addressing this issue immediately after the checkpoint.